<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputNodeInspector</class>
 <widget class="qMRMLWidget" name="InputNodeInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>311</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="qMRMLNodeComboBox" name="inputNodeSelector">
     <property name="nodeTypes">
      <stringlist notr="true">
       <string>vtkMRMLScalarVolumeNode</string>
       <string>vtkMRMLModelNode</string>
      </stringlist>
     </property>
     <property name="hideChildNodeTypes">
      <stringlist notr="true"/>
     </property>
     <property name="noneEnabled">
      <bool>true</bool>
     </property>
     <property name="addEnabled">
      <bool>false</bool>
     </property>
     <property name="removeEnabled">
      <bool>false</bool>
     </property>
     <property name="interactionNodeSingletonTag">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="dimensionsLabel">
     <property name="text">
      <string>Dimensions: -</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="spacingLabel">
     <property name="text">
      <string>Spacing: -</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="scalarRangeLabel">
     <property name="text">
      <string>Scalar range: -</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>InputNodeInspector</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
